README.md

```diff
@@ -20,6 +20,41 @@ The `@microsoft/fast-tooling` package contains a web worker referred to as the M
 
 The `@microsoft/fast-tooling-react` package contains various React components that work with the message system provided by `@microsoft/fast-tooling` to edit data, render data as HTML, and navigate data. To learn more, check out the package [README](./packages/fast-tooling-react).
 
+### Publishing
+This project uses [Beachball](https://microsoft.github.io/beachball/) to publish packages to NPM. The process is controlled through a series of commands located in the root `package.json` file to check, change, and publish. 
+
+When a change occurs within the configuration (`beachball.config.js`) parameters, Beachball will trigger interactive mode on the command line to capture additional details for generating the changelog file. The change instructions are saved to `./changes/*` folder and used during continuous delivery process on GitHub Actions (`.github/workflows/cd-publish-packages.yml`) to publish to NPM. 
+
+Beachball `publish:bump` can be used to version locally without publishing to the remote Git repository or NPM registry. This runs the same logic as `publish:start` so it's good practice to bump things locally to verify what's being changed.
```

Comment:
Is this something you want to advertise for any contributor to run? I'll be curious to see if this ends up helping or creating confusion. As a contributor I'd probably primarily care about my part in the publish process, this seems to elude to this being pertinent to it which I'm not sure it is.

Reply:
I would agree, see my comment above on extrapolating this into its own file.